projections/weather/lib/weather/shapes.rb

```diff
@@ -0,0 +1,92 @@
+# frozen_string_literal: true
+
+# This is generated code!
+
+module Weather
+  module Shapes
+    include Smithy::Client::Shapes
+
+    CityCoordinates = StructureShape.new(id: "example.weather#CityCoordinates", traits: {})
+    CityId = StringShape.new(id: "example.weather#CityId", traits: {"smithy.api#pattern"=>"^[A-Za-z0-9 ]+$"})
```

Comment:
Just a general comment/discussion on how we render hashes for things like this.  I ran into the same issue in the endpoint specs.  

This format is easy to write (since it relies on the default to_s method of a Hash), but produces code that is a (potentially very long) single line and violates rubocop rules (double quotes, missing spaces around brackets).  

We could create a utility for rendering these as multiline... or just leave it like this.  Thoughts?

Reply:
I think single line is fine for generated nonsense especially if this file will be very large.